arangopipe/Dockerfile_TFFE

```diff
@@ -8,14 +8,16 @@ COPY arangopipe_frontend/app/package.json /arangopipe_frontend
 COPY arangopipe_frontend/app/package-lock.json /arangopipe_frontend
 RUN npm install
 COPY arangopipe_frontend/app/ /arangopipe_frontend
-FROM tensorflow/tensorflow:latest-py3
+FROM tensorflow/tensorflow:1.14.0-py3
 
 MAINTAINER Joerg Schad <info@arangodb.com>
 ENV GIT_PYTHON_REFRESH=quiet
 RUN apt-get update
 RUN apt-get install -y python-pip
 RUN apt-get install -y python-pip curl
```

Comment:
Can we combine the apt get as well?

Reply:
@joerg84 will address this in the next push